FLEDGE_implementation_overview.md

```diff
@@ -0,0 +1,64 @@
+# Fledge Implementation Overview
```

Comment:
Thank you for this documentation Matt. 

Suggest you molt the FLEDGE refs.

Reply:
Done